### ISSUE TITLE
Convert to Composer Package

### ISSUE THREAD

**Issue by vrann** [colinmollenhour/Cm_Cache_Backend_Redis#93]
@colinmollenhour Would you be interested in converting the Redis cache backend to the composer package, so that Magento can avoid updating the framework in order to include new release of the backend? I think issues like this https://github.com/magento/magento2/pull/1231 should be handled by proper dependency management rather than by the code updates. Does this makes sense?

**Comment by colinmollenhour** [comment]
Yes, definitely makes sense and I am onboard with the idea in general. There is already a composer.json file though so I'm not sure what needs to be done to make it work with M2.
